pdc_client/__init__.py

```diff
@@ -187,10 +189,10 @@ def obtain_token(self):
 
     def get_paged(self, res, **kwargs):
         """
-        This call is equivalent to ``res(**kwargs)``, only it retrieves all pages
-        and returns the results joined into a single iterable. The advantage over
-        retrieving everything at once is that the result can be consumed
-        immediately.
+        This call is equivalent to ``res(**kwargs)``, if there is no self.page
```

Comment:
`page` should be passed as argument to this function. It's strange to have it common for all requests of one `PDCClient` instance.